packages/aws-cdk-lib/aws-scheduler/lib/schedule-expression.ts

```diff
@@ -0,0 +1,102 @@
+import { Construct } from 'constructs';
+import * as events from '../../aws-events';
+import { CronOptions } from '../../aws-events';
+import { Duration, TimeZone } from '../../core';
+
+
+/**
+ * ScheduleExpression for EventBridge Schedule
+ *
+ * You can choose from three schedule types when configuring your schedule: rate-based, cron-based, and one-time schedules.
+ * Both rate-based and cron-based schedules are recurring schedules.
+ *
+ * @see https://docs.aws.amazon.com/scheduler/latest/UserGuide/schedule-types.html
+ */
+export abstract class ScheduleExpression {
+  /**
+   * Construct a one-time schedule from a date.
+   *
+   * @param date The date and time to use. The millisecond part will be ignored.
+   * @param timeZone The time zone to use for interpreting the date. Default: - UTC
+   */
+  public static at(date: Date, timeZone?: TimeZone): ScheduleExpression {
+    try {
+      const literal = date.toISOString().split('.')[0];
+      return new LiteralScheduleExpression(`at(${literal})`, timeZone ?? TimeZone.ETC_UTC);
```

Comment:
I think the default should move to the `LiteralScheduleExpressions constructor` then is only appears **once** in the code.

Maybe it could also be put in a `const` or in a class somewhere?